Signal/src/util/OWSBackupIO.h

```diff
@@ -0,0 +1,52 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface OWSBackupEncryptedItem : NSObject
+
+@property (nonatomic) NSString *filePath;
+
+@property (nonatomic) NSData *encryptionKey;
```

Comment:
Per security review, we're now using a separate backup key for each uploaded file in a given backup.  The manifest is the only upload currently encrypted with the "backup encryption key".